jina/types/arrays/document.py

```diff
@@ -98,24 +107,48 @@ class DocumentArray(
     It gives an efficient view of a list of Document. One can iterate over it like
     a generator but ALSO modify it, count it, get item, or union two 'DocumentArray's using the '+' and '+=' operators.
 
-    :param doc_views: A list of :class:`Document`
-    :type doc_views: Optional[Union['RepeatedContainer', Iterable['Document']]]
+    It is supposed to act as a view containing a pointer to a `RepeatedContainer` of `DocumentProto` while offering `Document` Jina native types
+    when getting items or iterating over it
+
+    :param docs: the document array to construct from. One can also give `DocumentArrayProto` directly, then depending on the ``copy``,
+                it builds a view or a copy from it. It also can accept a List
+    :param copy: when ``docs`` is given as a :class:`DocumentArrayProto` object, build a
+                view (i.e. weak reference) from it or a deep copy from it.
```

Comment:
```suggestion
```
i dont think this is implemented anywhere

Reply:
if copy feature is not needed anywhere, then i suggest remove this docstring and ready to merge

Reply:
oh yes, leftover